.travis.yml

```diff
@@ -19,92 +20,22 @@ branches:
-    - www1.www.not-web-platform.test
-    - xn--n8j6ds53lwwkrqhv28a.www1.web-platform.test
-    - www2.not-web-platform.test
-    - www1.not-web-platform.test
-    - www1.www1.web-platform.test
-    - www2.xn--lve-6lad.web-platform.test
-    - www1.xn--n8j6ds53lwwkrqhv28a.not-web-platform.test
-    - xn--n8j6ds53lwwkrqhv28a.xn--n8j6ds53lwwkrqhv28a.web-platform.test
-    - www1.www2.web-platform.test
-    - xn--lve-6lad.xn--n8j6ds53lwwkrqhv28a.not-web-platform.test
-    - xn--lve-6lad.xn--lve-6lad.not-web-platform.test
-    - www2.www1.not-web-platform.test
-    - www2.xn--n8j6ds53lwwkrqhv28a.web-platform.test
-    - www.xn--n8j6ds53lwwkrqhv28a.not-web-platform.test
-    - web-platform.test
-    - xn--n8j6ds53lwwkrqhv28a.xn--lve-6lad.not-web-platform.test
-    - xn--lve-6lad.www.web-platform.test
-    - www.www.web-platform.test
-    - www1.xn--lve-6lad.web-platform.test
-    - xn--lve-6lad.www2.web-platform.test
-    - www.www2.web-platform.test
-    - www2.www.web-platform.test
-    - xn--n8j6ds53lwwkrqhv28a.www2.not-web-platform.test
-    - www1.www.web-platform.test
-    - www.xn--lve-6lad.web-platform.test
-    - www.www.not-web-platform.test
-    - xn--lve-6lad.xn--lve-6lad.web-platform.test
-    - xn--lve-6lad.www1.not-web-platform.test
-    - www1.www1.not-web-platform.test
-    - xn--n8j6ds53lwwkrqhv28a.xn--n8j6ds53lwwkrqhv28a.not-web-platform.test
-    - xn--n8j6ds53lwwkrqhv28a.www2.web-platform.test
-    - xn--n8j6ds53lwwkrqhv28a.www1.not-web-platform.test
-    - www2.www2.web-platform.test
-    - xn--lve-6lad.web-platform.test
-    - xn--n8j6ds53lwwkrqhv28a.web-platform.test
-    - xn--n8j6ds53lwwkrqhv28a.www.web-platform.test
-    - xn--lve-6lad.not-web-platform.test
-    - www2.xn--n8j6ds53lwwkrqhv28a.not-web-platform.test
-    - www2.xn--lve-6lad.not-web-platform.test
-    - xn--n8j6ds53lwwkrqhv28a.xn--lve-6lad.web-platform.test
-    - www1.xn--lve-6lad.not-web-platform.test
-    - www1.web-platform.test
-    - www.www1.web-platform.test
-    - www2.web-platform.test
-    - www.xn--lve-6lad.not-web-platform.test
-    - www.web-platform.test
-    - xn--lve-6lad.xn--n8j6ds53lwwkrqhv28a.web-platform.test
-    - www1.www2.not-web-platform.test
-    - www2.www1.web-platform.test
-    - www.www1.not-web-platform.test
-    - www.xn--n8j6ds53lwwkrqhv28a.web-platform.test
-    - xn--lve-6lad.www2.not-web-platform.test
-    - www2.www.not-web-platform.test
-
 matrix:
   include:
     - node_js: 8
       env: TEST_SUITE=node-canvas
-      script: "export CXX=g++-4.8 && yarn add canvas && yarn test --retries 1"
-    # - node_js: stable
-    #   env: TEST_SUITE=node-canvas
-    #   script: "export CXX=g++-4.8 && yarn add canvas && yarn test --retries 1"
+      addons:
+        apt:
+          packages:
+            - build-essential
+            - libcairo2-dev
+            - libpango1.0-dev
+            - libjpeg-dev
+            - libgif-dev
+            - librsvg2-dev
+      script: "yarn add canvas && yarn test --retries 1"
```

Comment:
Nice cleanup with the packages and no longer requiring the CXX environment variable